src/test/java/com/sonyericsson/jenkins/plugins/bfa/TransientCauseManagementHudsonTest.java

```diff
@@ -64,7 +65,7 @@
     public void testOnAProject() throws Exception {
         FreeStyleProject project = createFreeStyleProject("nils-job");
         project.getBuildersList().add(new Shell("env | sort"));
-        project = configRoundtrip(project);
+        project = (FreeStyleProject)configRoundtrip((Item)project);
```

Comment:
Had to do this here (and below) to workaround eclipse reporting "The method configRoundtrip is ambiguous for the type..." after freshly cloning the repository, like here https://github.com/jenkinsci/jenkins/commit/cabb5f010a9c450c4604f693644b685791f0fac5. Can get rid of it if requested.